Section 1: Introduction and Setup/4-RoR Development Environment/testing_project/README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
`README.md` is a markdown text file used by owner or contributors to specify more info regarding the repository. This could include info on how to work with the files, important remarks on certain changes, pictures of the working demo, license of the project and many more. This is the first thing you see displayed when you clicked into a repository.